src/js/models/v2/BaseFederalAccountAwardRow.js

```diff
@@ -19,12 +19,9 @@ const BaseFederalAccountAwardRow = {
         this.awardingToptierAgency = data.awarding_agency.toptier_agency.name || '';
         this.awardingSubtierAgency = data.awarding_agency.subtier_agency.name || '';
         this._issuedDate = parseDate((data.latest_transaction && data.latest_transaction.action_date));
-        this._loanValue = (data.latest_transaction
-            && data.latest_transaction.assistance_data
-            && data.latest_transaction.assistance_data.face_value_loan_guarantee) || 0;
+        this._loanValue = data.total_subsidy_cost || 0;
         this._subsidyCost = (data.latest_transaction
-            && data.latest_transaction.assistance_data
-            && data.latest_transaction.assistance_data.original_loan_subsidy_cost) || 0;
+            && data.latest_transaction.original_loan_subsidy_cost) || 0;
```

Comment:
This and `total_subsidy_cost` look like strings, should we do `parseFloat` on them?